media_driver/agnostic/gen11/codec/kernel_free/build.py

```diff
@@ -0,0 +1,199 @@
+import os,shutil
+import sys
+
+import struct
+import os
+from sys import platform
+
+class GenSource2Bin(object):
+    def __init__(self, component='codec'):
+        self.fromdir    = './Binary'
+
+    def iga_compile(self):
+        for filename in os.listdir('./Source'):
+            fp = os.path.join('./Source', filename)
+            fileext = os.path.splitext(filename)[1]
+            if os.path.isfile(fp) and 'cpp' in fileext:
+                command  = 'cmc /c /Qxcm -Qxcm_jit_target=gen11lp '
```

Comment:
-Qxcm_jit_target=gen11lp - why?! The kernel is inside gen11 folder. So, for which platform this kernel is compiled?